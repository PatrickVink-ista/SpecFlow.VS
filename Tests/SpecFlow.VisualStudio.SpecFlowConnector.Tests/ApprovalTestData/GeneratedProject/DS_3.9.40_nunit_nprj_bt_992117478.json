{
    "GeneratorOptions": {
        "featureFileCount": 1,
        "scenarioPerFeatureFileCount": 1,
        "newProjectFormat": true,
        "targetFramework": "net48",
        "SpecFlowPackageVersion": "3.9.40"
    }
}
