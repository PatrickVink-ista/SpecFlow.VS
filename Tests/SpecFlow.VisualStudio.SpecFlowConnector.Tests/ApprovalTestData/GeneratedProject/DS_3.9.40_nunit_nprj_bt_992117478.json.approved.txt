{
  "stepDefinitions": [
    {
      "type": "Then",
      "regex": "^lorem dictum (\\d\u002B) accumsan$",
      "method": "ConubiaDonecEgestasQuisqueSteps.ThenAnteCursusJustoId(Int32)",
      "paramTypes": "i",
      "expression": "lorem dictum (\\d\u002B) accumsan",
      "sourceLocation": "#0|10|9|12|10"
    },
    {
      "type": "When",
      "regex": "^nulla nec dui (\\d\u002B)$",
      "method": "ConubiaDonecEgestasQuisqueSteps.WhenSemDonecCommodoEt(Int32)",
      "paramTypes": "i",
      "expression": "nulla nec dui (\\d\u002B)",
      "sourceLocation": "#0|16|9|18|10"
    },
    {
      "type": "Given",
      "regex": "^id nulla (\\d\u002B)$",
      "method": "ConubiaDonecEgestasQuisqueSteps.GivenConubiaDonecDiamEleifend(Int32)",
      "paramTypes": "i",
      "expression": "id nulla (\\d\u002B)",
      "sourceLocation": "#0|22|9|24|10"
    }
  ],
  "sourceFiles": {
    "0": "<<targetFolder>>\\StepDefinitions\\ConubiaDonecEgestasQuisqueSteps.cs"
  },
  "typeNames": {}
}
